src/packets/incoming/mapinfo-packet.ts

```diff
@@ -107,6 +113,7 @@ export class MapInfoPacket implements Packet {
     for (let i = 0; i < this.extraXML.length; i++) {
       this.extraXML[i] = reader.readStringUTF32();
     }
+    this.gameOpenedTime = reader.readInt32();
```

Comment:
It says `readUnsignedInt()` in Flash client so it should probably be `reader.readUInt32()` here.